css/fonts.css

```diff
@@ -0,0 +1,19 @@
+@font-face {
+font-family: 'Gotham Pro Regular';
+src: url('../fonts/GothamPro.eot');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)